mypy/stubtest.py

```diff
@@ -362,6 +362,10 @@ class SubClass(runtime):  # type: ignore
                 runtime_attr = getattr(runtime, mangled_entry)
             except AttributeError:
                 runtime_attr = inspect.getattr_static(runtime, mangled_entry, MISSING)
+            if runtime_attr is MISSING and issubclass(runtime, enum.Enum):
+                members = getattr(runtime, "__members__", [])
+                if members:
+                    runtime_attr = getattr(members[0], mangled_entry, MISSING)
```

Comment:
Let's please describe why we use 0th member.

Reply:
There's no good reason for using the 0th member, but there's no good reason for using any other member either. What do you propose I say in the comment? :)

Reply:
Do we assume they are all identical? If so, we just use the one that 100% exists 🤔